apps/examples/board_specific_demo/ameba_secure_world_demo/Kconfig

```diff
@@ -6,7 +6,7 @@
 config EXAMPLES_SECURE_WORLD
 	bool "Secure World example"
 	default n
-	depends on ARMV8M_TRUSTZONE && SE_AMEBA
+	depends on (ARMV8M_TRUSTZONE || AMEBASMART_TRUSTZONE) && SE_AMEBA
```

Comment:
`ARMV7M_TRUSTZONE` has no relation with `CONFIG_AMEBASMART_TRUSTZONE`.
Please add `selects ARMV7M_TRUSTZONE` to AMEBASMART_TRUSTZONE like AMEBALITE_TRUSTZONE.

Reply:
Sorry, I mean `ARMV7A`, not `ARMV7M`.